### DISCUSSION TITLE
Should we archive the repo ?

### DISCUSSION THREAD

**Discussion by Apollinaire** [VulcanJS/Vulcan#2767] [Announcements]
It's all in the title really, most of us maintainers have moved to something else and the only issues/notifications we get now are kind of irrelevant or just npm package updates. Archiving the repo(s) 

@SachaG @eric-burel @ErikDakoda Any objections ?

I suggest we also archive all the vulcan-something repos, to make things clear.

**Comment by ErikDakoda** [comment]
I am fine with that.

**Comment by eric-burel** [comment]
Yep it's an extinct Vulcan. Hope you folks are doing fine! For googlers that may end up here, most of the codebase is now integrated within the State of JS survey monorepo: https://github.com/Devographics/Monorepo

**Reply by eric-burel** [reply]
@SachaG I don't have the permission level to archive repos on my side

**Reply by Apollinaire** [reply]
@eric-burel I think I have the required permissions to archive.
Do you want to write a message telling people how the project evolved to devographics, and redirecting them ?

**Reply by eric-burel** [reply]
Yep that would be nice thank you! I'll edit the READMEs later on with a message indeed
